src/synchronizeContacts.spec.js

```diff
@@ -1,6 +1,6 @@
 const synchronizeContacts = require('./synchronizeContacts')
 
-describe('synchronizeContacts function', () => {
+describe.skip('synchronizeContacts function', () => {
```

Comment:
Did you mean to keep this skipped?

Reply:
no, this test was not very useful so I added tests that check the real use case using the strategies in 5ab4654  and 994d49f